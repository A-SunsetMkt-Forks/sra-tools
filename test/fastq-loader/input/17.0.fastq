@0f950b71-118b-4c7d-8030-4610650687c2_Basecall_1D_template
UGCCCUGAUUCAUGAUCCCCGACGAUGGGGGGGCAAUUUUGAGGAGUUCCAUCUGUGGUUCCUACAAGAACUCGGGCCUUGAAACCUGGCCUAUGGCACUUCGAGAAGCAUAUCCUAUCUCAAGAACACAGUUUCCCCAUUUUGUGUGUGGUUCAAGUUCUGGUCAGUGAGAAACUCCAUGACCACAAAGUUUACCACGUCGGUAGUCAACUCCUUAAGUGGGAUCUUCCUUUAUGAUGAAACCAUACAAGUAUCGGUUUUCGCCUCACCCUCUUAGUUCUACUUGCUUAUCAUUGGAAUUAAAUUGGAAAUCAUCCAUUGCUGCACAAGAUGCUACUAAGUAAGGCAGUAUUGAUGUACACCCCUUAUGUAUUUUGUGGGCUCUGCAGUGAGACACCUCUGAUGUAAUUCCUCAUGCUUGCAGCGUGUCAAUUGGAGCCACAUCUCCAAUAAUAUCCAGCUUGAAAUUUGGGUCAGUUCGCAUGCACCAUUGUUGUUGAACUCAUUCUGAAUCCAUUCUUGCAAGCGACCUCAAUUCUGGUCUCAACCCAUUAUAUUGCUUCAAUUCUCCUAUUCAUCAGUCGUUUAAGUCUGGUGCCAUCGUUCUUAAGUGCCCACUUUGGUAUCCGUUCCACUCCCCCCUUUAGUUUUGGAAUUUUUCCUUCAUUUUUGUAUCCUGCAGUUUGCCAGUACUUGUUCAUGACAGAAGAUAAUUUGAUUUAUUCCCUUCUAUGGUUUAACAACAUUGGGUUCCAUCCAAAGAAUGUUCAUGCAUUUGAUUGCAUCAUAGCAGUAUUCCCUCUCCCUCCACAUGACUUUGGUCCUCAAUGCUUAAUUUUUGGGCAUCCAUCAGCAGGGAAUUUGGUCGCUGAGAACAGGAGGCCCAUGGAAGUCUAAGUGGUCUGGUGACUGCUUUCAAAAAGGUCGCUAGCAUUUACUUUGGACAUUUGAGACAGCUUGCCCCUUGUAGCUGCUUCGGUUCGAAUCCAUCCAUGGGCUCAUAAUACAAAGGCUAGAAAGUCUCGGCGAGACUUUGGUCGGCAAGCUUGCGCAUUGUUCCUGUGAUCUCAAACCUUCAAUAUUGUUCUGAAGACAAAGAUGAAAGAAAUACAAGUUGUUGGCCAUGGUUACGGUGAAUAGCCAAAGUUAAGAUCUAGCCAAAGUUCCUAUGAGAGUGAGUAGCCUGUGGGCCAGUGAACGAGAAAAAUGUGGAUGUGUGUGUUUCAGAUUUAAAUUUUAUUAUGGCCUUUCCCAGAUGGUAUGUGAACUUCCUGUUACUCCAAUUUCGAUGAAUCUAUUUUCCUGUAAUAUACAAAUCUGGUAGAAACUUGUUUCAGCCCAGCAGUGUUGCAAAAUUGGCACUACUAGUCCAGGCCAUUGUGCGAUCUUCCCAUGAUUAUUCAAAUCUGUGUGCUUUCAAAGUGCUAUUAGGAUCACCAAGUUCUACGAUUAUUGACUCGCCUUAGCUCUAUUGAUGAAGUAGAAUCUGAAUACUGUAGUAUUUUCUAGUGGAGUGCUAUUGCUGCAAAUUUGUUUUUCGAUUUUCAGGUCCCAUACUUUAUUGUUUUUUUUCAAGUCGACAAUUAUCGGAUUGAAGCAUUGUUGCAGCUACUUUGAUUUGAUCUGCUUUUGCUAUCUCCCAUCCCAUCCCGUUUUAUCCAUAUCAUCAACCAUCCCAUCCCUAUCUUCCUCAUAU
+
#"""######""##"###%##""#"####"#$%"$%'+20%$'(#,)+.+%%"(%'*-0./+*%%&&$)*'%%+42*.&,)3<04:(**+**((&5&'%&)(&/4740.)((%()'&)*$&&0(.5021*+,(&$+1.+*661.-,*,+/,-(+6+)++'*,+**)2,*)().2+,11..-%(**+//)-+.+&('&'&&'(,'$-'&(.'+)0)1*-+++34,(%(''&*0-*+036/=>9060.))()(('($+4)+-(%()+%'$'03$%*-)/*-$((''))3;60.)(+-,+.-,,.4+1/-./6*)&--1-+1./5885/-12.12/()+*0)/(&%&*+,,+,)*.)*--*(%#&063&((++03,0++(*-72&&#%""&**,+++,-7--).-+*))+.1)*)%%$%)%+',('*'***.+,1902/+-01''&&%)**0+%%3&(*513-*01-,2..5./+,(%&.+,)*'(***),*&'*().&&0-/,))*(-/)-3:320)-''%/51/1/-++),.-/7.-*&/+1*()'(*'(&'&+''%(+-),-%'0(+$'%)%&&)$$'$%%&)).&%.-)%%%#$,,,.''$$&'$(&(-(&(*420),6/)-(#%('''*'*##''.693.#%(&(+*.-)+.)+11*)(,'+('(1/*$$(*+2201/.+2)..54+)(((0*)%%'$$'''(*00.--*3+845--&++-/)3())$&#$)))/+/,+3-++-../0+*-*('+-+4..78-,,0.-.13110,,*+-+)1--)$+0/01.0)*26,&&'*+*%)+&)++,/)*-,1+*4--14654++--4(,,,.(2,))+01/0,,+*)+7,4.0'//*&+))('-).1*%)*$'.*5'&&$&/1)*).----.++&)*,(*(,1/$00+-,583+/,****320.*-,)(&,,3(5)))*8300.+,*))-3),8;,+$&%#'(',*(&)+,+--.72/1/-)1)&$'./('$$&++(%&()-',(,-.31,)($)%&,+*,+'$--1+720*2+.1--/2,++*'&,*-,/.-,010.'*-2=393**%&'%%**,&'%&#()((('%%%&''/3*-)1/$(*..**,*+*,**&$%#%+(###'&(+.$%$&*-5,++3)/+**'(-(+.'+.*.'''#&''*((('&#$%#%&)&('),,$%%%%))43-/+29;7,+**2/.,+)(')+*0-*'''23/7:84,.'*%&)%+0,-/+0-)2.('+*&**())++2.-,+**).//,/+.2,.*,)(+./..6+*%%&/+)0+1012.,((),+55,,/.7--,/<+(,*&')('&'695%)%*+,,/653496,*$$&(''()(*'+'&+&(2*1*.+*',(,+,%('(.)%)++-4*.0),=7,'.-,0,1%-),&%&$#**(+('&))3**+*0217('/$&'((+,.)+-((%&''**()$%%'#&%&*'*+,.*(%&(-2'&**1/..'%$##(%&%()(%$%)()'%/++)**+.,)551-41(-**'*+-.%/18//-+(.,'-6,*&&%&'(%,*-*23337-(%#$&%%$%%&&-)''$**+11-1/,,,/1/02+,*'((%$$&&$%((&(&'')),+644.312*$$--(%$'-&##&(&#&((*$&%&&&'%%''((*&'$$*.*('+/,'+(&&$&*#%$###
