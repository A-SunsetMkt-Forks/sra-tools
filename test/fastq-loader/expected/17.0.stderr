latf-load err: data incomplete while reading data within alignment module - Premature EOF, FASTQ_input failed
latf-load warn: The file contained no records that were processed.
latf-load err: data empty while reading file within alignment module - accession="obj" errors="0" status="failure"
latf-load err: data empty while reading file within alignment module - load failed
